pmd-core/src/main/java/net/sourceforge/pmd/lang/ast/impl/antlr4/AntlrToken.java

```diff
@@ -17,9 +17,13 @@
  */
 public class AntlrToken implements GenericToken<AntlrToken> {
 
-    private final Token token;
```

Comment:
I removed this so that we don't store two different image strings for each token, which would be memory inefficient.